packages/synapse-react-client/src/components/JsonSchemaForm/templates/BaseInputTemplate.tsx

```diff
@@ -34,7 +34,8 @@ export default function BaseInputTemplate<
     disabled,
     type,
     label,
-    // hideLabel,
+    hideLabel: _hideLabel,
+    hideError: _hideError,
```

Comment:
Prevent these from being included in the `...textFieldProps` spread below. These were passed to a component that didn't recognize them, causing a warning to be thrown in dev/test